sklearn/lda.py

```diff
@@ -1,6 +1,20 @@
 import warnings
+from .discriminant_analysis import LinearDiscriminantAnalysis as _LDA
```

Comment:
For future references: please don't accept my PR comments without explanation. They are as likely as not to be bs ;)